.github/workflows/test.yml

```diff
@@ -6,12 +6,20 @@ on:
   schedule:
     - cron: '0 6 * * *'
 
+defaults:
+  run:
+    shell: bash
```

Comment:
I guess that would place windows tests into msys2 shell. Would be great to test under as well, but it would no longer be "native windows"

Reply:
This setting only affects the shell used to interpret `run:` actions in the workflow.  It doesn't affect any command invocations within the script.

Reply:
it might.  E.g. in non-msys2 environment you would have different PATH variable and thus possibly use different tools than if running "natively"